static/js/home.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('#join-us')
+let buttonPosition = (button.offsetTop - 15)
+
+function adaptButtonPosition() {
+  buttonPosition = (button.offsetTop - 15)
+  return buttonPosition
+}
+
+function stickyJoinButton(event) {
+  
+  let diff = (event.pageY-buttonPosition)
+  
+  if (diff > 0) {
+    button.setAttribute('class', 'sticky'); 
+  } else {
+    button.removeAttribute("class", "sticky"); 
+  }
```

Comment:
You could replace these 5 lines with:

```js
button.classList.toggle('sticky', diff > 0)
```

https://developer.mozilla.org/en/docs/Web/API/Element/classList#Methods